mecanum_drive_controller/test/diff_drive_test.cpp

```diff
@@ -0,0 +1,151 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Hello @efernandez, I have actually fixed the code according to your review. Thanks for your help ;)